### DISCUSSION TITLE
Is it possible to mark a section that original text be reserved?

### DISCUSSION THREAD

**Discussion by catplanet007** [yuin/goldmark#234] [Q&A]
I use hugo, markdown file first tranformed by goldmark, then rendered by katex.

I found my math equation in markdown not rendered well, annd then I found that the output of goldmark was not what I expected.

Here is my code:

```go
package main

import (
	"bytes"

	"github.com/yuin/goldmark"
)

func main() {
	var buf bytes.Buffer
	if err := goldmark.Convert([]byte(`$$\Omega=\{\omega\}$$`), &buf); err != nil {
		panic(err)
	}
	println(buf.String()) // print: <p>$$\Omega={\omega}$$</p> 
}
```

`\{` was tranformed to `{`, and `\}` was tranformed to `}`, 

I've tried at [CommonMark online demo](https://spec.commonmark.org/dingus/), and nothing different from this output.

So I was asking how can I set `$$` and `$` as separators to mark a section where original text would be reserved?

I‘ve browsed through all the extensions and none of them seemed to work for my situation.

**Comment by night-watchman** [comment]
Did you find any way to solve this problem？I have the same problem.
